integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/WithKyuubiServerOnKubernetes.scala

```diff
@@ -17,17 +17,22 @@
 
 package org.apache.kyuubi.kubernetes.test
 
+import scala.collection.mutable
+
 import io.fabric8.kubernetes.client.DefaultKubernetesClient
 
 import org.apache.kyuubi.WithKyuubiServer
 import org.apache.kyuubi.config.KyuubiConf
 
 trait WithKyuubiServerOnKubernetes extends WithKyuubiServer {
   protected val kyuubiServerConf: KyuubiConf = KyuubiConf()
-  protected val connectionConf: Map[String, String]
+  protected var connectionConf: mutable.Map[String, String] = mutable.Map.empty[String, String]
```

Comment:
why do we need this change ..

Reply:
Well, I'm not very familiar with Scala maps, and I'm not sure if += is possible with maps

Reply:
We need to build duplicate configurations in `KyuubiOnKubernetesWithSparkOnKubernetesTestsBase`, and then build special configurations in `Client test` and `Cluster test`, so we chose not to use a pure Map.